content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -547,6 +547,19 @@ This annotation requires the [NodePreferAvoidPods scheduling plugin](/docs/refer
 to be enabled. The plugin is deprecated since Kubernetes 1.22.
 Use [Taints and Tolerations](/docs/concepts/scheduling-eviction/taint-and-toleration/) instead.
 
+### preemption-toleration.scheduling.sigs.k8s.io/minimum-preemptable-priority
+
+Example: `preemption-toleration.scheduling.sigs.k8s.io/minimum-preemptable-priority: "10000"`
+
+This specifies the minimum priority value that can preempt this [PriorityClass](/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass). It defaults to the PriorityClass's priority value + 1 if not set, which means pods that have a higher priority value can preempt it.
```

Comment:
- Is this an annotation or a label?
- What kind of objects is this used on?

Reply:
- This is an annotaion, you also mentioned it while opening the issue. Should i add something like "This annotation specifies ___" to make it more clear.
- This is used on PriorityClass object, then the PriorityClass object is used for defining pod configuration. So, 
`Used On: PriorityClass` sound accurate to me.

Reply:
> Should i add something like "This annotation specifies ___" to make it more clear.

Yes

You should add Used On text too; see other entries for an example.